viewer/packages/camera-manager/src/Flexible/IFlexibleCameraManager.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { FlexibleControlsOptions } from './FlexibleControlsOptions';
+import { CameraManager } from '../CameraManager';
+
+/**
+ * Flexible implementation of {@link CameraManager}. The user can switch between Orbit, FirstPersion or OrbitInCenter
+ * Supports automatic update of camera near and far planes and animated change of camera position and target.
+ * @beta
+ */
+export interface IFlexibleCameraManager extends CameraManager {
+  get options(): FlexibleControlsOptions;
```

Comment:
I'd ideally like us to return an interface to the options here, if for nothing else, at least so that we can more flexibly switch out the implementation.